SONAR-netCDF4/docs/tablePlatform.adoc

```diff
@@ -152,27 +157,44 @@ a|
 |:units = "m/s" | |
 |float :valid_min = 0.0 | |
 | | |
-|float transducer_offset_x |R |Distance from the platform coordinate system origin to the monostatic sonar transducer along the _x_-axis.
-|:long_name =  +
-"_x_-axis distance from the +
+|float transducer_offset_x(transducer) |R |Distance from the platform coordinate system origin to the transducer along the _x_-axis.
+|:long_name = "_x_-axis distance from the +
 platform coordinate system +
 origin to the sonar transducer" | |
 |:units = "m" | |
 | | |
-|float transducer_offset_y |R |Distance from the platform coordinate system origin to the monostatic sonar transducer along the _y_-axis.
-|:long_name =  +
-"_y_-axis distance from the +
+|float transducer_offset_y(transducer) |R |Distance from the platform coordinate system origin to the transducer along the _y_-axis.
+|:long_name = "_y_-axis distance from the +
 platform coordinate system +
 origin to the sonar transducer" | |
 |:units = "m" | |
 | | |
-|float transducer_offset_z |R |Distance from the platform coordinate system origin to the monostatic sonar transducer along the _z_-axis.
-|:long_name =  +
-"_z_-axis distance from the +
+|float transducer_offset_z(transducer) |R |Distance from the platform coordinate system origin to the transducer along the _z_-axis.
+|:long_name = "_z_-axis distance from the +
 platform coordinate system +
 origin to the sonar transducer" | |
 |:units = "m" | |
 | | |
+|int transducer(transducer) |R |transducer serial number
```

Comment:
Not all transducers have a serial number that is a number (some have letters). Suggest to change this to:

```suggestion
|int transducerId(transducer) |R |Transducer serial number or identification name
```

Reply:
```suggestion
|string transducerId(transducer) |R |Transducer serial number or identification name
```
Just a pip misplaced